test/run_test.py

```diff
@@ -0,0 +1,245 @@
+"""Test retrace-server before make install.
+
+run: python run_test.py [--coredump="/path/to/the/coredump"
+        --executable="executable" --package="package"]
+        [--delete_repo] [--dont_create_repo]
+        --release="Fedora release 24 (Twenty Four)"
+When coredump path is passed, executbale and package are required
+When no coredump path is passed, coredump will be generated and executable
+   and package are ignored
+Warning: using delete_repo will remove repo, even though it was not created
+        by this script, be careful if you use your own repo
+Note: might need sudo privileges due to using /var/spool/
+"""
+
+from __future__ import print_function
+import os
+import argparse
+import sys
+import shutil
+sys.path.append("./../src")
```

Comment:
True. I thought PYTHONPATH env is used in the Makefile. Please, remove the path configuration out from the test and point python to correct directory using PYTHONPATH.

```bash
PYTHONPATH=${abs_top_srcdir}/src
```